aztec-up/bin/docker-compose.yml

```diff
@@ -32,5 +32,7 @@ services:
       PXE_BLOCK_POLLING_INTERVAL_MS: 50
       ARCHIVER_VIEM_POLLING_INTERVAL_MS: 500
       PXE_PORT: ${PXE_PORT:-8080}
+      PORT:  ${SANDBOX_PORT:-8080}
```

Comment:
Should `PXE_PORT` and `PORT` default to the same thing?

Reply:
they do currently so yes